src/openapi/parameters/query/query.yml

```diff
@@ -1,5 +1,6 @@
 name: query
 in: query
+allowEmptyValue: true
```

Comment:
Because of this line, do you mean that the `nullable: true` underneath is not useful ?